mk/macros.mk

```diff
@@ -0,0 +1,20 @@
+ifndef post-manifests
+define post-manifests
+@[ -e .venv ] || $(MAKE) .venv
+@(source .venv/bin/activate; yaml2json config/rbac/role.yaml) | jq '.kind="Role"' > config/rbac/role.json \
+&& (source .venv/bin/activate; json2yaml config/rbac/role.json > config/rbac/role.yaml) \
+&& rm -f config/rbac/role.json \
+&& (source .venv/bin/activate; yaml2json config/rbac/role_binding.yaml) | jq '.kind="RoleBinding" | .roleRef.kind="Role"' > config/rbac/role_binding.json \
+&& (source .venv/bin/activate; json2yaml config/rbac/role_binding.json) > config/rbac/role_binding.yaml \
+&& rm -f config/rbac/role_binding.json
```

Comment:
This is called at 'make manifests' to impact the less as possible into the Makefile that generate `operator-sdk`. 

This update the ClusterRole and ClusterRoleBindings for the controller manager to Role and RoleBindings resources to restrict the controller permissions to a namespaced scope instead of cluster wide. When the manifests are generated.